Yoast/wordpress-seo
# Add tests for the indexable presenters

## Context


* We want to have full coverage of the code in `src/presenters`.

## Summary


This PR can be summarized in the following changelog entry:

* N/A Adds tests for the presenters.

## Relevant technical choices:

* ~~For `get_id()` in the `breadcrumbs_presenter`, I didn't test the scenario where the ID is already set, because currently the ID doesn't get set in the `breadcrumbs_presenter`, and it's hard to set up this scenario because `id` is a private variable.~~
    * ~~The same applies to `get_class()`, `get_wrapper()`, `get_separator()` and `get_element()`.~~
* 👆This is fixed in the last [commit](https://github.com/Yoast/wordpress-seo/pull/14488/commits/0d258fa5bb91618639f70ea7fac97bfb2d3edcde): we now call the tests twice to test caching. If the caching doesn't work correctly, the tests will fail since the filters are expected to be called only `once`.
* For `present()` in the `meta_description_presenter`, I didn't test the scenario where the meta description is not a string, because the `trim()` function always returns a string.
* I added missing documentation in various places.
* I removed the URL helper from `twitter/image-presenter` because it wasn't used.

## Test instructions

This PR can be tested by following these steps:

* See all the unit tests pass.
* See the code coverage in `src/presenters` is at 100%.

## UI changes

## Documentation

## Quality assurance


Fixes #
